website/content/docs/release-notes/v0_19_0.mdx

```diff
@@ -199,5 +213,23 @@ description: >-
     </td>
   </tr>
 
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+    0.19.0
+    <br /><br />
+    (Fixed in 0.19.1)
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+    Deleted auth tokens prevent reauthentication
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+    In previous versions, when the auth tokens associated with a user were deleted, Boundary deleted the user as well. When the user logged in later, Boundary would have to reload the user's resources before they could access them.
+    <br/><br />
+    Version 0.19.1 introduces a soft delete for when a cached user's auth tokens are deleted and there is a valid refresh token that is less than 20 days old. Boundary marks the user as deleted, so they do not show up in lists and searches. But it keeps their user information in the cache.
+    <br /><br />
+    When the user logs in again, Boundary changes the user's status to active and there is no need to reload the resources. When the refresh token is older than 20 days, Boundary deletes the user and their resources from the cache.
```

Comment:
I *think* there's some rewording needed here for accuracy? 

Version 0.19.0 introduced the concept of a "soft delete" in the client cache, but there was an issue where "soft deleted" users weren't being restored when the user logs back in again. Version 0.19.1 addresses that issue. This was causing search functionality to break.

cc: @johanbrandhorst

Reply:
Thanks @moduli ! I reread the comments on #5549, and I think you are correct. I just pushed an updated version of this entry.